src/components/Confirm/index.js

```diff
@@ -21,12 +21,17 @@ const FileViewer = ({file:{preview, type}}) =>
     </embed>
   </object>
 
-
-const CaptureViewerPure = ({capture:{blob}}) =>
+const CaptureViewerPure = ({capture:{blob, base64}}) =>
```

Comment:
I will keep `capture:{}` as parameter since it's easier to reason that it's one capture with multiple versions this way.